src/main/java/com/launchdarkly/client/RedisFeatureStore.java

```diff
@@ -245,26 +274,64 @@ public boolean initialized() {
     return getInit();
   }
 
+  /**
+   * Releases all resources associated with the store. The store must no longer be used once closed.
+   * @throws IOException
+   */
+  public void close() throws IOException
+  {
+    pool.destroy();
+  }
+
+
 
   private String featuresKey() {
     return prefix + ":features";
   }
 
   private Boolean getInit() {
-    return jedis.exists(featuresKey());
+    Jedis jedis = null;
```

Comment:
Not a big priority, but if we switch to java 7 or 8 this can eliminate a lot of boilerplate in this changeset:
http://docs.oracle.com/javase/7/docs/technotes/guides/language/try-with-resources.html